symfony/ux-meta/1.0/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+    "bundles": {
+        "Symfony\\WebpackEncoreBundle\\WebpackEncoreBundle": ["all"]
```

Comment:
ux-meta should not register WebpackEncoreBundle (especially not if `symfony/ux-meta` does not directly require the bundle)